gatorgrade/main.py

```diff
@@ -0,0 +1,35 @@
+"""Use Typer to run gatorgrade to run the checks and generate the yml file."""
+from pathlib import Path
+import typer
+
+app = typer.Typer(add_completion=False)
+FILE = "gatorgrade.yml"
+
+
+@app.callback(invoke_without_command=True)
+def gatorgrade(
+    filename: Path = typer.Option(
+        f"{FILE}", "--config", "-c", help="Name of the yml file."
```

Comment:
This format string is not needed--it can just be `FILE`, instead of `f"{FILE}"`.